### DISCUSSION TITLE
Vim : moving between pane and adding new keybinding

### DISCUSSION THREAD

**Discussion by laplantePierre** [zed-industries/zed#13643] [Feature Requests]
> If if move to terminal with ˆwj, how do I move back to editor from terminal

> Can you add new keymap in zed as in vim:
Keymap('n', '<leader>z', 'a<C-R>=strftime("%Y-%m-%d")<CR> user@domainéco,<Esc>', opts)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
